src/modules/tests/tests.analytics.effects.ts

```diff
@@ -69,4 +74,32 @@ export class TestsAnalyticsEffects {
       return of();
     }),
   );
+
+  @Effect()
+  testOutcomeChangedEffect$ = this.actions$.pipe(
```

Comment:
Missing tests - will add them in my own pr as I've already added a new test class for this

Reply:
Cool, thanks man!